src/map/MapSidebar.js

```diff
@@ -97,6 +126,138 @@ function MapSidebarMain (props) {
+      if (isMountedNominal) setNominalIncome(" "+data);
+    });
+    return () => {isMountedNominal = false; };
+
+  },[nominalIncome, setNominalIncome,getCeremonyIncome, api, cid])
+
+  const [registeredBootstrappers, setregisteredBootstrappers]= useState([]);
+  const [registeredReputables, setregisteredReputables]= useState([]);
+  const [registeredEndorseees, setregisteredEndorsees]= useState([]);
+  const [registeredNewbies, setregisteredNewbies]= useState([]);
+  const [unassignedNewbies, setunassignedNewbies]= useState([]);
+  
+// gets the data of all people that registered for a Ceremony
+  function showPhasepeopleData(){
+    useEffect(() => {
+      async function getPhasepeopleData(){
+        const currentCeremonyIndex = await api.query.encointerScheduler.currentCeremonyIndex();
+        const currentCommunityCeremony = new CommunityCeremony(api.registry, [cid, currentCeremonyIndex])
+        const [tempregisteredBootstrappers, tempregisteredReputables, tempregisteredEndorseees,tempregisteredNewbies, assignmentCounts] = await Promise.all([
+          api.query.encointerCeremonies.bootstrapperCount(currentCommunityCeremony),
+          api.query.encointerCeremonies.reputableCount(currentCommunityCeremony),
+          api.query.encointerCeremonies.endorseeCount(currentCommunityCeremony),
+          api.query.encointerCeremonies.newbieCount(currentCommunityCeremony),
+          api.query.encointerCeremonies.assignmentCounts(currentCommunityCeremony)
+        ]);
+        return [tempregisteredBootstrappers, tempregisteredReputables, tempregisteredEndorseees, tempregisteredNewbies, assignmentCounts]
+      }
+
+      let isMounted = true;
+      getPhasepeopleData().then((data) => {
+        if(isMounted){
+        setregisteredBootstrappers(data[0]);
+        setregisteredReputables(data[1]);
+        setregisteredEndorsees(data[2]);
+        setregisteredNewbies(data[3]);
+        setunassignedNewbies(registeredNewbies-data[4].newbies);
+        }
+      })
+      return () => {isMounted = false; }; 
+
+    },[api, cid]);
+    if(currentPhase==0){
+      return (<div>
+        <li>The registered Bootstrapper count is: {registeredBootstrappers.toString()}</li>
+        <li>The registered Reputables count is: {registeredReputables.toString()}</li>
+        <li>The registered Endorsees count is: {registeredEndorseees.toString()}</li>
+        <li>The registered Newbies count is: {registeredNewbies.toString()}</li>
+      </div>);
+    }else{
+      return (<div>
+        <li>The registered Bootstrapper count is: {registeredBootstrappers.toString()}</li>
+        <li>The registered Reputables count is: {registeredReputables.toString()}</li>
+        <li>The registered Endorsees count is: {registeredEndorseees.toString()}</li>
+        <li>The registered Newbies count is: {registeredNewbies.toString()}</li>
+        <li color='red'>The number of unassigned Newbies is: {unassignedNewbies.toString()}</li>
+      </div>);
+    }
```

Comment:
Here I would reduce the amount of redundant information. Please write something like:

Registered participants for this ceremony:

* Bootstrappers: 5
* Reputatbes: 45
* Endorsees: 10
* Newbies: 10
* Unassigned Newbies: 0